ext/data/recommended-dictionaries.json

```diff
@@ -1,13 +1,25 @@
 {
     "ja": {
         "terms": [
-            {"name": "Jitendex", "url": "https://github.com/stephenmk/stephenmk.github.io/releases/latest/download/jitendex-yomitan.zip"}
+            {
+                "name": "Jitendex",
+                "url": "https://github.com/stephenmk/stephenmk.github.io/releases/latest/download/jitendex-yomitan.zip",
+                "description": "An improved version of JMdict for Yomitan. It features better formatting and some other improvements, and is actively being improved by its author."
+            }
         ],
         "kanji": [
-            {"name": "KANJIDIC", "url": "https://github.com/themoeway/jmdict-yomitan/releases/latest/download/KANJIDIC_english.zip"}
+            {
+                "name": "KANJIDIC",
+                "url": "https://github.com/themoeway/jmdict-yomitan/releases/latest/download/KANJIDIC_english.zip",
+                "description": "An English dictionary listing readings, meanings, and other info about kanji characters."
```

Comment:
Can detail the more specific features of KANJIDIC (stroke order diagrams, statistics, JLPT level etc..)
yomitan.wiki docs is unfortunately not very descriptive :( yet